src/test/java/com/google/javascript/clutz/tsickle_emit.skip.tsickle.js

```diff
@@ -0,0 +1,7 @@
+goog.module('a.path.to.the.module');
+
+class KlosureKlass {
```

Comment:
If I remember correctly, we had some code that iterates through types used by other code, and still emits those.

Can you add a test for a situation where you have another file `emitted.js` that references a type from `tsickle_emit.skip.tsickle.js`?

Reply:
Ah, right. Fixed with https://github.com/angular/clutz/pull/574.